reference-architecture/gce-cli/ansible/playbooks/roles/rhel-image/defaults/main.yaml

```diff
@@ -0,0 +1,9 @@
+---
+rhel_image_path: '{{ ansible_env.HOME }}/Downloads/rhel-guest-image-7.3-35.x86_64.qcow2'
```

Comment:
That's fine. Once it passes travis Ill give it one more look and then merge or ping you. Thanks!!